### PR TITLE
Quote1

### PR DESCRIPTION
**Pull request by aqk**

### CODE REVIEW

**Comment by aqk** [comment]
Note that this won't pass tests without https://github.com/Chia-Network/clvm/pull/51

File: stages/stage_0.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 def run_program(
     program,
     args,
-    quote_kw=KEYWORD_TO_ATOM["q"],
+    quote_kw=KEYWORD_TO_ATOM["quote"],
```
Comment (arvidn) [inline stages/stage_0.py:12]: I see, the quote op-code *is* configurable now too. shouldn't this use `QUOTE_KW` rather than a hard coded "quote"?

Comment (arvidn) [reply stages/stage_0.py:12]: actually, shouldn't this parameter name be `quote_atom` and shouldn't the default value be `QUOTE_ATOM`?

File: stages/stage_0.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
     apply_kw=KEYWORD_TO_ATOM["a"],
```
Comment (arvidn) [inline stages/stage_0.py:13]: here too, it seems we're not following the naming convention of `_KW` being the *name* and `_ATOM` being the *value*/*op-code*

**Comment by arvidn** [comment]
looks reasonable. I had a few comments and I'm generally uneasy about the sweeping change of `q` -> `quote`, but I'm sure there's a good reason

**Pull request closed without merging**